pg_shard.c

```diff
@@ -625,8 +664,7 @@ PlanSequentialScan(Query *query, int cursorOptions, ParamListInfo boundParams)
 	enable_indexscan = false;
 	enable_bitmapscan = false;
 
-	queryCopy = copyObject(query);
-	sequentialScanPlan = standard_planner(queryCopy, cursorOptions, boundParams);
+	sequentialScanPlan = standard_planner(query, cursorOptions, boundParams);
```

Comment:
So the reason we kept this here is because the standard_planner scribbles on the query input. This way, I know PlanSequentialScan will keep my inputs unchanged. Removing this means the caller needs to be aware of this, and create a copy of the query in advance before calling this function. I'd be in favor of keeping it here.